server/test/api/employee.js

```diff
@@ -114,7 +114,46 @@ describe('The /employees API endpoint :: ', function () {
       .catch(helpers.handler);
   });
 
-  it('PUT /employee/:id should update an existing employee ', function () {
+  it('METHOD : GET PATH : /employees/:key/:value,  should return a list of employees match the employee code ', function () {
```

Comment:
Instead of writing `:/key/:value` it would be best to write exactly what you are executing.  These are messages for future developers to read and understand what is happening.